extensions-contrib/opentelemetry-extensions/pom.xml

```diff
@@ -42,11 +42,6 @@
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
         </dependency>
-        <dependency>
-            <groupId>io.opentelemetry.proto</groupId>
-            <artifactId>opentelemetry-proto</artifactId>
-            <version>${opentelemetry.proto.version}</version>
-        </dependency>
```

Comment:
you still need to declare the dependency, but you would omit the version now. The dependency-management section in the parent pom doesn't declare the dependency, it only defines the default version and scope for downstream modules if they choose to depend on it.

Reply:
yea, just figured that from failed compilation. 😂 thanks!

Reply:
fixed!